docs/source/tutorials/portfolio_optimization.ipynb

```diff
@@ -0,0 +1,485 @@
+{
```

Comment:
<p>typo: 'thereby ensuring consistency with the classical approach' -&gt; 'which is consistent with what we found with classical brute force search.'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tencent-quantum-lab/tensorcircuit/pull/186/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>